resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientApplication.scala

```diff
@@ -145,7 +157,17 @@ private[spark] class Client(
     try {
       val otherKubernetesResources = resolvedDriverSpec.driverKubernetesResources ++ Seq(configMap)
       addOwnerReference(createdDriverPod, otherKubernetesResources)
-      kubernetesClient.resourceList(otherKubernetesResources: _*).createOrReplace()
+      var commonResources = Seq.empty[HasMetadata]
```

Comment:
This is for owner reference updated for pre created resource, we can only update owner reference after pod ready, becasuse we need the pod info like pod uid